terra_layer/views/layers.py

```diff
@@ -37,21 +34,23 @@ class LayerViewset(ModelViewSet):
     model = Layer
     serializer_class = LayerSerializer
     ordering_fields = (
-        "source",
-        "group",
         "name",
+        "source__name",
+        "group__view__name",
+        "active_by_default",
         "order",
+        "in_tree",
+    )
+    filter_fields = (
+        "source",
+        "group",
+        "active_by_default",
+        "in_tree",
         "table_enable",
         "popup_enable",
         "minisheet_enable",
-        "group__view__name",
     )
-    filter_fields = ("source",)
     permission_classes = ()
-    filter_backends = api_settings.DEFAULT_FILTER_BACKENDS + [
-        filters.SearchFilter,
-        URLFilterBackend,
-    ]
```

Comment:
this will disable search and url filtering, isn't it ? Or it is specified in REST_FRAMEWORK project settings ?

Reply:
it won't disable it, it will default to `REST_FRAMEWORK` setting so it must be set by the project using this package, rather than this package being opinionated and coupling to either `django-filter` or `url-filter`.

Reply:
ok